AUTHORS.md

```diff
@@ -0,0 +1,19 @@
+EasyRPG Player authors
+======================
+
+* Alejandro Marzini (vgvgf)
+* Alessio Tosto (Rinnegatamante)
+* Carsten Teibes (carstene1ns)
+* Christian Breitwieser (ChrisBreiti)
+* Diego Pedraza (zegeri)
+* Dmytro Kushnariov (rohkea)
+* Francisco de la Peña (fdelapena)
+* Gabriel Kind (ghabry)
+* Glynn Clements (glynnc)
+* Khelifi Mehdi (BlisterB)
+* Mariano Javier Suligoy (MarianoGNU)
+* Patrick Müssig (Tondorian)
+* Paulo "Zhek" Vizcaino (paulo_v)
+* scurest
```

Comment:
blame @fdelapena (https://github.com/EasyRPG/Player/commit/0332ec111c618119e14b805364040373f3c8fa6b) :dancer: yeah, will change it later.